packages/monitoring/src/helpers/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestError } from "@threefold/types";
+import axios, { AxiosError } from "axios";
+import { ServiceStatus } from "src/types";
+
+export async function sendGetRequest(url: string, headers?: Record<string, string>) {
+  const options = {
+    method: "get",
+    url: url,
+    headers: headers || {},
+  };
+  try {
+    return await axios(options);
+  } catch (e) {
+    const { response } = e as AxiosError;
+    let errorMessage = (response?.data as { error: string })?.error;
```

Comment:
this should have been more concise e.response?.data.error || e.message I believe